File: atinject-core/src/main/java/org/atinject/api/session/dto/SessionOpenedNotification.java
package org.atinject.api.session.dto;

import org.atinject.core.websocket.dto.WebSocketNotification;

public class SessionOpenedNotification extends WebSocketNotification
{
    private static final long serialVersionUID = 1L;
    
    public String sessionId;
    
    public SessionOpenedNotification(){
        super();
    }

    public String getSessionId()
    {
        return sessionId;
    }

    public SessionOpenedNotification setSessionId(String sessionId)
    {
        this.sessionId = sessionId;
        return this;
    }
    
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/session/SessionOpenedNotification.js
define(["atinject/WebSocketNotification"],
function(WebSocketNotification){
    function SessionOpenedNotification() {
        this._class = "org.atinject.core.session.dto.SessionOpenedNotification"
    };

    SessionOpenedNotification.prototype = new WebSocketNotification();

        SessionOpenedNotification.prototype.getSessionId = function() {
        return this.sessionId;
    };

    SessionOpenedNotification.prototype.setSessionId = function(sessionId) {
        this.sessionId = sessionId;
        return this;
    };


    
    return SessionOpenedNotification;
});
